linear.cpp

```diff
@@ -2809,6 +2810,14 @@ struct model *load_model(const char *model_file_name)
 	model *model_ = Malloc(model,1);
 	parameter& param = model_->param;
 
+        param.C = 1;
+        param.p = .1;
+        param.eps = INF;
+        param.nr_weight = 0;
+        param.weight_label = NULL;
+        param.weight = NULL;
+        param.init_sol = NULL;
+
```

Comment:
agreed - spaces vs tabs.  i've fixed it by tabifying the file (and my editor automatically removed trailing spaces)